libs/api/src/services/admin/types.ts

```diff
@@ -41,6 +41,21 @@ export const ElectionRecordSchema: z.ZodSchema<ElectionRecord> = z.object({
   isOfficialResults: z.boolean(),
 });
 
+/**
+ * Info related to a CVR file import attempt.
+ */
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type CvrFileImportInfo = {
+  id: Id;
+  alreadyPresent: number;
+  exportedTimestamp: Iso8601Timestamp;
+  fileMode: CvrFileMode;
+  fileName: string;
+  newlyAdded: number;
+  scannerIds: string[];
+  wasExistingFile: boolean;
+};
```

Comment:
```suggestion
/**
 * Info related to a CVR file import attempt.
 */
export interface CvrFileImportInfo {
  id: Id;
  alreadyPresent: number;
  exportedTimestamp: Iso8601Timestamp;
  fileMode: CvrFileMode;
  fileName: string;
  newlyAdded: number;
  scannerIds: string[];
  wasExistingFile: boolean;
}
```

Reply:
my type-fu might be lacking here, but i hit an issue where the compiler refused to accept this line unless i define this as `type` instead of `interface`:

```
export type PostCvrFileResponse =
  | OkResponse<CvrFileImportInfo>
  | ErrorsResponse;
```

https://github.com/votingworks/vxsuite/pull/2780/files#diff-184a93dafa7bda07c33812aba5e31c48efcaa8fd42ab3a75a150bd755084b79aR182

might be that i need to tweak `OkResponse`'s definition a bit to get this working - any tips?

also, for context: what's our reasoning for preferring `interface` over `type` in vxsuite?

Reply:
oh, i misremembered where the error was occurring - it's a little further downstream when i try using the schema for `PostCvrFileResponse`:

https://github.com/votingworks/vxsuite/blob/main/frontends/election-manager/src/lib/backends/admin_backend.ts#L330

```
[election-manager:build] src/lib/backends/admin_backend.ts(345,7): error TS2345: Argument of type 'ZodType<PostCvrFileResponse, ZodTypeDef, PostCvrFileResponse>' is not assignable to parameter of type 'ZodType<ErrorsResponse | OkResponse<Record<string, unknown>>, ZodTypeDef, ErrorsResponse | OkResponse<Record<string, unknown>>>'.
[election-manager:build]   Type 'PostCvrFileResponse' is not assignable to type 'ErrorsResponse | OkResponse<Record<string, unknown>>'.
[election-manager:build]     Type 'OkResponse<CvrFileImportInfo>' is not assignable to type 'ErrorsResponse | OkResponse<Record<string, unknown>>'.
[election-manager:build]       Type 'OkResponse<CvrFileImportInfo>' is not assignable to type 'OkResponse<Record<string, unknown>>'.
[election-manager:build]         Type 'OkResponse<CvrFileImportInfo>' is not assignable to type 'Record<string, unknown>'.
[election-manager:build]           Index signature for type 'string' is missing in type 'OkResponse<CvrFileImportInfo>'
```

gonna give it another shot, tweaking the types around the response and `fetchWithSchema` and see if i can get it to work with the interface declaration 🤞

Reply:
no luck so far - will table this for now to avoid spinning for too long, but down to revisit and fix later